### ISSUE TITLE
Use Symfony Serializer 

### ISSUE THREAD

**Issue by tacman** [dmishh/SettingsBundle#127]
I think the Symfony Serializer might be better than php's serialize().

Or in addition to, I guess.

**Comment by MartinCigarsky** [comment]
Is it okay if try to implement this? I also will provide the performance tests.
@rvanlaak

**Comment by rvanlaak** [comment]
Yes sure, feel free to. Always great to hear willingness to contribute 💪
